docs/src/operations/requirements.md

```diff
@@ -55,13 +63,17 @@ made.
 - CPU
   - 16 cores / 32 threads, or more
 - RAM
-  - 512 GB or more if `account-index` is used
+  - 512 GB or more if an `account-index` is used, 1TB+ for all three [account indexes](https://docs.solanalabs.com/operations/setup-an-rpc-node#account-indexing)
 - Disk
   - Consider a larger ledger disk if longer transaction history is required
   - Accounts and ledger should not be stored on the same disk
 
 ## Virtual machines on Cloud Platforms
 
+**NOTE**: Running on cloud platforms is **not recommended** (especially on
+mainnet) due to excessive egress costs and bad co-tenants starving your node of
+resources necessary to stay in sync. The info in this section is outdated.
```

Comment:
Shouldn't we remove the outdated section instead of saying "the info below is outdated"?

Reply:
yes. could probably replace with something like
```
running a solana node in the cloud requires significantly greater
operational expertise to achieve stability and performance. do not
expect to find sympathetic voices should you chose this route and
find yourself in need of support.
```

Reply:
Nuked the cloud section and replaced with suggested text in cf8d6dbe7891746c9b46d8f173d9424687961d72